MovieChecker/MovieChecker/BoxOfficeCell.h

```diff
@@ -0,0 +1,17 @@
+//
+//  BoxOfficeCell.h
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/22/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface BoxOfficeCell : UITableViewCell
+
+@property (nonatomic, weak) IBOutlet UILabel *nameLabel;
+@property (nonatomic, weak) IBOutlet UILabel *salesLabel;
+@property (nonatomic, weak) IBOutlet UIImageView *thumbnailImageView;
```

Comment:
Changed these here, but in BoxOfficeCollectionViewCell, it seems that I need to expose thumbnailImageView, in order to work with SDWebImage, unless I want to move the image downloading method calls to the Cell's implementation, which I probably don't.  (I might also be able to make a dummy UIImageView, and set the cell's image to match, but I don't know if that's a good idea.)